albulaUtils.py

```diff
@@ -87,14 +90,31 @@ def _albulaDispFile(filename):
         if not (currentMasterH5 == filename[0]):
             logger.info('reading file: %s' % filename[0])
             albulaSubFrame.loadFile(filename[0])
-            imgSeries = dectris.albula.DImageSeries(filename[0])
             currentMasterH5 = filename[0]
+            sleep(0.3) # Sleep to allow Albula to load file. Otherwise the following goTo() is ignored
         logger.debug('reading image number %s' % filename[1])
-        albulaSubFrame.loadImage(imgSeries[filename[1]])
+        albulaSubFrame.goTo(filename[1])
     except dectris.albula.DNoObject:
         albulaFrame = dectris.albula.openMainFrame()
         albulaSubFrame = albulaFrame.openSubFrame()
         imgSeries = dectris.albula.DimageSeries(filename[0])
         albulaSubFrame.loadImage(imgSeries[filename[1]])
         currentMasterH5 = filename[0]
-
+    except Exception as e:
+        logger.error(f'Albula exception: {e}')
+
+def validate_master_HDF5_file(filename):
+  """
+  Check if H5 file is valid 
```

Comment:
for now, validation consists of checking whether the data files exist and are readable. could you update this comment? I expect we could do more validation later on, so would like to keep the function name as it is

Reply:
Added comment in 2ad40c0445b7ce7facec643c18bb611089f3d74e

Reply:
looks good now. resolving